bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -554,7 +554,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- ================================================================== -->
 
 <!-- <fates_paramfile>lnd/clm2/paramdata/fates_params_api.36.1.0_14pft_c241003.nc</fates_paramfile> -->
-<fates_paramfile>lnd/clm2/paramdata/fates_params_api.36.1.0_14pft_c241003_megan.nc</fates_paramfile>
+<fates_paramfile>lnd/clm2/paramdata/fates_params_api.36.1.0_14pft_c250131_megan_drydep.nc</fates_paramfile>
```

Comment:
@mvdebolskiy it seems to me that the COMPARE tests are failing (for FATES-SP and NOCOMP, without drydep) ONLY when I include the change to the parameter file here. Can you think of any reason why that might be?

Reply:
Ahhh. That might be it yes. At least, the file it was pointing to before might have had that switch flipped... That would make it much less mysterious. Thanks :)